patches/0023-Provide-context-to-conns.patch

```diff
@@ -27,27 +27,138 @@ index b6e19a7283..9176fa0a16 100644
  	registryID := d.Get("registry_id").(string)
  	log.Printf("[DEBUG] Reading ECR repository credentials %s", registryID)
 diff --git a/internal/service/elasticache/security_group.go b/internal/service/elasticache/security_group.go
-index f587fcbcce..f77ca2b249 100644
+deleted file mode 100644
+index f587fcbcce..0000000000
 --- a/internal/service/elasticache/security_group.go
-+++ b/internal/service/elasticache/security_group.go
-@@ -58,7 +58,7 @@ func resourceSecurityGroupCreate(ctx context.Context, d *schema.ResourceData, me
- 
- func resourceSecurityGroupRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
- 	var diags diag.Diagnostics
++++ /dev/null
+@@ -1,127 +0,0 @@
+-package elasticache
```

Comment:
I'm assuming they added this upstream so we don't need it anymore?

Reply:
So this code started conflicting or not compiling but I think it was dead code. We didn't link this resource anywhere in Pulumi. Instead of fixing the code I deleted it.

Reply:
ahh makes sense